### PR TITLE
GODRIVER-1481 Update godoc link to pkg.go.dev

### PR DESCRIPTION
**Pull request by houjunchen**

### CODE REVIEW

**Comment by houjunchen** [comment]
The check failed because I don't have the access to evergreen.
If this is an inappropriate pull request, please let me know. I will close it.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <p align="center"><img src="etc/assets/mongo-gopher.png" width="250"></p>
 <p align="center">
   <a href="https://goreportcard.com/report/go.mongodb.org/mongo-driver"><img src="https://goreportcard.com/badge/go.mongodb.org/mongo-driver"></a>
-  <a href="https://godoc.org/go.mongodb.org/mongo-driver/mongo"><img src="etc/assets/godoc-mongo-blue.svg" alt="GoDoc"></a>
-  <a href="https://godoc.org/go.mongodb.org/mongo-driver/bson"><img src="etc/assets/godoc-bson-blue.svg" alt="GoDoc"></a>
+  <a href="https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo"><img src="etc/assets/godoc-mongo-blue.svg" alt="GoDoc"></a>
```
Comment (divjotarora) [inline README.md]: In `godoc-mongo-blue.svg` and `godoc-bson-blue.svg`, can you change the word "godoc" to something else (e.g. "docs") as we're not using GoDoc anymore?

**Comment by divjotarora** [comment]
@houjunchen Renaming the files is not quite sufficient. If you go to https://github.com/mongodb/mongo-go-driver/blob/master/etc/assets/godoc-bson-blue.svg and click on "Raw", you can see that the file actually contains the word "godoc". This is what causes the word "godoc" to appear in the button. Can you change that to "docs" for both that file and `godoc-mongo-blue.svg`?

**Comment by houjunchen** [comment]
@divjotarora how about this: https://github.com/houjunchen/mongo-go-driver/tree/godoc

**Comment by divjotarora** [comment]
This looks good. I've authorized the Evergreen build and I'll check back in ~30 minutes to approve/merge the PR if the build looks good.

**Comment by divjotarora** [comment]
@houjunchen I'm going to merge this PR and close the relevant GODRIVER ticket. Once again, thank you for your contribution to the driver!

**Pull request merged**